### ISSUE TITLE
Documentation for `datalad.get.subdataset-source-candidate` option gives an incorrect property for URL of a configured remote

### ISSUE THREAD

**Issue by mslw** [datalad/datalad#7458]
### Current docs
The [documentation for datalad get](http://docs.datalad.org/en/stable/generated/man/datalad-get.html) (section: getting subdatasets) says:

> Additional candidate URLs can be generated based on templates specified as configuration variables with the pattern
>
>    `datalad.get.subdataset-source-candidate-<name>`
>
> (...)
>
> Additionally, the URL of any configured remote that contains the respective submodule commit is available as `remote-<name>` properties, where NAME is the configured remote name.

### Current code
However, the code of `_get_flexible_source_candidates_for_submodule()` caters for `remoteurl-<name>` property, not `remote-<name>`:

https://github.com/datalad/datalad/blob/928498eb91fed29e0f56fc84a0ca2197fd647e4d/datalad/distribution/get.py#L207-L210

### Practical example

Tested in practice, where `../foo/bar` is the url of the origin remote:

- With source candidate `{remote-origin}/{path}` given 000 priority it doesn't work:

```
[WARNING] Failed to format template '{remote-origin}/{path}' for a submodule clone. Error: KeyError('remote-origin') 
[DEBUG  ] Assembled 2 clone candidates for bar: ['https://github.com/mslw/foo-bar.git', '../foo/bar']
```

With source candidate `{remoteurl-origin}/{path}` given 000 priority it works, and moves origin ahead of the URL from `.gitmodules`:

```
[DEBUG  ] Assembled 2 clone candidates for bar: ['../foo/bar', 'https://github.com/mslw/foo-bar.git']
```

<hr>

Discovered thanks to [this Neurostars question](https://neurostars.org/t/how-to-set-clone-locations-to-local-only-on-hpc/26370?u=mszczepanik)

**Comment by mslw** [comment]
Oh, and the docstring for `_get_flexible_source_candidates_for_submodule` lists totally different values (590 / 600 / 650 / 900) than the docstring for `get` - the former seems to be right :thinking:

**Comment by adswa** [comment]
note to self: need to check [this handbook chapter](https://handbook.datalad.org/en/latest/beyond_basics/101-148-clonepriority.html) if there is wrong information, too.

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`0.19.3`](https://github.com/datalad/datalad/releases/tag/0.19.3)
